core/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -81,6 +81,8 @@
  */
 public abstract class TransportReplicationAction<Request extends ReplicationRequest, ReplicaRequest extends ReplicationRequest, Response extends ActionWriteResponse> extends TransportAction<Request, Response> {
 
+    public static final String REPLICATION_ACTION_SHARD_FAILURE_TIMEOUT = "cluster.replication.action.shard.failure_timeout";
```

Comment:
we typically follow the package name as setting prefix. So `action.support.replication` here.